File: iniciante/javascript/b1035.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let line1 = lines.shift().split(' ');
let A = parseInt(line1[0]);
let B = parseInt(line1[1]);
let C = parseInt(line1[2]);
let D = parseInt(line1[3]);

if (B > C && C + D > A + B && C > 0 && D > 0 && A % 2 === 0) {
  console.log('Valores aceitos');
} else {
  console.log('Valores nao aceitos');
}

File: iniciante/swift/b1035.swift
import Foundation

var values = readLine()!.split(separator: " ")
let a = Int(values[0])!
let b = Int(values[1])!
let c = Int(values[2])!
let d = Int(values[3])!

if ((b > c) && (d > a) && (c + d > a + b) && (c > 0) && (d > 0) && (a % 2 == 0)) {
  print("Valores aceitos")
} else {
  print("Valores nao aceitos")
}